nototools/unittests/font_tests.py

```diff
@@ -330,25 +330,55 @@ def test_format_4_cmap_preset(self):
 
 
 class TestLigatures(FontTest):
-    """Tests formation or lack of formation of ligatures."""
+    """Tests formation or lack of formation of ligatures.
+
+    Must have the following attributes:
+        active: List of (parameters, (sequence, sequence, ...)) tuples, for
+            which each sequence should form a ligature when shaped with the
+            given parameters. The ligature should not be formed by default if
+            parameters are provided.
+        inactive: Similar list of tuples, for sequences which should form a
+            ligature under inverse circumstances.
```

Comment:
I'm not quite sure I get what 'inverse circumstances are' without it being spelled out.  It sounds like, if parameters are provided, a ligature should form with all sets of parameters except those provided (including the default case), and if parameters are not provided, a ligature should form with all sets of parameters except the default case.